api_docgen/Format_tutorial.mld

```diff
@@ -247,6 +247,20 @@ break hint that precedes a closing parenthesis can show the boxes structure, if
 it leads to a new line; hence "\[(---\[(----\[(---b)\]b)\]b)\]" is printed like
 this:
 
+{[
+(---
+ (----
+  (---
+   )
+  )
+ )
+]}
+
+However the closing parenthesis don't line up with the opening ones, to
+correctly show the structure we need to introduce additional structural
+boxes after the parenthesis.
```

Comment:
This is not an explanation of what it is happening: the issue is that the closing parenthesis is indented with respect to the indentation of box containing the break hints.

If one want to align the closing parenthesis with the opening one, we can either cancel the box indentation with a negative indentation, or indeed add one more level of box with no indentation.

Reply:
I've included something similar in `doc(format): mention 2 alternatives for getting structural boxes that line up vertically`, and added you as `Suggested-by` on the commit.